### PR TITLE
Work on styling and edit moment bug

### PR DESCRIPTION
**Pull request by jpanopio**

### CODE REVIEW

File: client/app/components/capture.js [hunk 1]
```diff
@@ -79,46 +73,19 @@ var styles = StyleSheet.create({
...
 
 var cameraRollStyles = StyleSheet.create({
   row: {
     flexDirection: 'row',
-    flex: 1,
-  },
-  url: {
-    fontSize: 9,
-    marginBottom: 14,
+    width: 130,
```
Comment (AlexDWu) [inline client/app/components/capture.js:81]: magic!

File: client/app/components/editMoment.js [hunk 2]
```diff
@@ -126,8 +130,9 @@ var EditMoment = React.createClass({
         })
         .then((response) => response.json())
         .then((responseData) => {
+          console.log('responseData: ', responseData);
```
Comment (AlexDWu) [inline client/app/components/editMoment.js]: THE CONSOLE THE LOG!

File: client/app/components/capture.js [hunk 3]
```diff
@@ -10,32 +10,24 @@ import React, {
...
+import externalStyles from '../style/external-styles.js';
 
 
 const CAMERA_ROLL_VIEW = 'camera_roll_view';
 
 class Capture extends Component {
-
   _renderImage(asset) {
-    const imageSize = 150;
+    const imageSize = 120;
```
Comment (AlexDWu) [inline client/app/components/capture.js:20]: magic

Comment (suyax) [inline client/app/components/capture.js:20]: magic number

File: client/app/components/editMoment.js [hunk 4]
```diff
@@ -146,9 +150,9 @@ var EditMoment = React.createClass({
                 };
               })
               .then((result) => {
+                console.log(result);
```
Comment (AlexDWu) [inline client/app/components/editMoment.js:153]: THE CONSOLE LOG!

Comment (suyax) [inline client/app/components/editMoment.js:153]: console.log()

File: client/app/components/home.js [hunk 5]
```diff
@@ -182,15 +175,20 @@ var styles = StyleSheet.create({
...
     textAlign: 'center',
-    color: ' white',
-    margin: 5,
+    fontSize: 10,
+    color: ' #5379ae',
+    marginRight: 4,
   },
+  textBox: {
+    flexDirection: 'column',
+    width: 120,
```
Comment (AlexDWu) [inline client/app/components/home.js:186]: magic

**Pull request merged**